CITATION.cff

```diff
@@ -1,6 +1,7 @@
-cff-version: 0.5.0
+cff-version: 1.2.0
```

Comment:
This was incorrect, should be schema version here
https://github.com/citation-file-format/citation-file-format

Reply:
Added version field below :)